network/endpoint_linux.go

```diff
@@ -53,205 +54,200 @@ func (nw *network) newEndpointImpl(
 	nl netlink.NetlinkInterface,
 	plc platform.ExecClient,
 	netioCli netio.NetIOInterface,
-	epClient EndpointClient,
-	epInfo *EndpointInfo,
+	testEpClient EndpointClient,
```

Comment:
why was this renamed?

Reply:
renamed since we have may use multiple epClients when looping through epInfo now

looks like non-nil epClient is only passed during testing

Reply:
since this is only non-nil during test

Reply:
added comment